src/main/java/edu/hm/hafner/analysis/parser/AquaScannerParser.java

```diff
@@ -0,0 +1,92 @@
+    private static final long serialVersionUID = 1L;
+
+    @Override
+    protected void parseJsonObject(final Report report, final JSONObject jsonReport, final IssueBuilder issueBuilder) {
+        JSONArray resources = jsonReport.optJSONArray("resources");
+        if (resources != null) {
+            parseResources(report, resources, issueBuilder);
+        }
+    }
+
+    private void parseResources(final Report report, final JSONArray resources, final IssueBuilder issueBuilder) {
+        for (int i = 0; i < resources.length(); i++) {
+            JSONObject resourceWrapper = (JSONObject) resources.get(i);
+            if (!resourceWrapper.isNull("vulnerabilities") && !resourceWrapper.isNull("resource")) {
+                JSONObject resource = resourceWrapper.getJSONObject("resource");
+                JSONArray vulnerabilities = resourceWrapper.getJSONArray("vulnerabilities");
+                for (Object vulnerability : vulnerabilities) {
+                    report.add(convertToIssue(resource, (JSONObject) vulnerability, issueBuilder));
+                }
+            }
+        }
+    }
+
+    private Issue convertToIssue(final JSONObject resource, final JSONObject vulnerability,
+            final IssueBuilder issueBuilder) {
+        final String fileName = resource.isNull("path") ? resource.optString("name", VALUE_NOT_SET)
+                : resource.getString("path");
+        return issueBuilder
+                .setFileName(fileName)
+                .setSeverity(mapSeverity(vulnerability.optString("aqua_severity", "UNKNOWN")))
+                .setMessage(vulnerability.optString("name", "UNKNOWN"))
+                .setDescription(formatDescription(fileName, resource, vulnerability))
+                .buildAndClean();
+    }
+
+    @SuppressFBWarnings("IMPROPER_UNICODE")
+    private Severity mapSeverity(final String string) {
+        if (AQUA_VULNERABILITY_LEVEL_TAG_LOW.equalsIgnoreCase(string)
+                || AQUA_VULNERABILITY_LEVEL_TAG_NEGLIGIBLE.equalsIgnoreCase(string)) {
+            return Severity.WARNING_LOW;
+        }
+        else if (AQUA_VULNERABILITY_LEVEL_TAG_MEDIUM.equalsIgnoreCase(string)) {
+            return Severity.WARNING_NORMAL;
+        }
+        else if (AQUA_VULNERABILITY_LEVEL_TAG_HIGH.equalsIgnoreCase(string)
+                || AQUA_VULNERABILITY_LEVEL_TAG_CRITICAL.equalsIgnoreCase(string)
+                || AQUA_VULNERABILITY_LEVEL_TAG_MALWARE.equalsIgnoreCase(string)
+                || AQUA_VULNERABILITY_LEVEL_TAG_SENSITIVE.equalsIgnoreCase(string)) {
+            return Severity.WARNING_HIGH;
+        }
+        else {
+            return Severity.WARNING_HIGH;
+        }
+    }
+
+    private String formatDescription(final String fileName, final JSONObject resource,
+            final JSONObject vulnerability) {
+        return MessageFormat.format(
+                "<p><div><b>Resource</b>: {0}</div><div><b>Installed Version:</b> {1}</div><div><b>Aqua Severity:</b> {2}</div>",
```

Comment:
I'm fine with coding this in text. But I am wondering why you are using `j2html` in the `Descriptor` but not here, this looks not consistent (while it creates the same result).

Reply:
Changed  to use j2html. 
Original was grabbed from [TrivyParser](https://github.com/jenkinsci/analysis-model/blob/d4e9d1f9cb7130732151cbff69a238c0fa54ba62/src/main/java/edu/hm/hafner/analysis/parser/TrivyParser.java#L90) and it was using unclosed `<p>` tag (not sure if it makes any difference).

### Before (String.format):

```html
<p>
<div><b>Resource</b>: bla.txt</div>
<div><b>Installed Version:</b> 1.0.0</div>
<div><b>Aqua Severity:</b> HIGH</div>
<p>detected bad vulnerability</p>
```

![image](https://user-images.githubusercontent.com/27741935/163066713-6f5c51f8-8788-475a-a176-c2b867393c7b.png)


### After (j2html):

```html
<div><b>Resource: </b>bla.txt</div>
<div><b>Installed Version: </b>1.0.0</div>
<div><b>Aqua Severity: </b>HIGH</div>
<p>detected bad vulnerability</p>
```

![image](https://user-images.githubusercontent.com/27741935/163066736-953e806a-e507-4931-8a0d-734c8910d08e.png)